landing-pages/site/content/en/blog/documenting-using-local-development-environments.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Documenting using local development environment"
+linkTitle: "Documenting using local development environment"
+author: "Elena Fedotova"
+github: "efedotova"
+linkedin: "elena-fedotova-039294110"
+description: "The story behind documenting local development environment of Apache Airflow"
+tags: ["Development"]
+date: 2019-11-22
+---
+
+## Documenting local development environment of Apache Airflow
+
+From Sept to November, 2019 I have been participating in a wonderful initiative, Google Season of Docs.
```

Comment:
Can you add link to initiative website?

Reply:
do you mean GSOD? https://developers.google.com/season-of-docs

Reply:
I can't edit this md file(

Reply:
Yes. Link to GSOD will be useful.

Reply:
I will do it @efedotova :)

Reply:
I would still like the links to be in the middle of the text, and not just at the very end. Do you think that makes sense? This would allow easier access to them.  This is not necessary for me.

Reply:
Let me know if you want to correct this or merge a change otherwise.

Reply:
But I think it's just natural flow of the narrative.  I don't even know how to change it so that it is in the middle of paragraph :D.